assets/global.js

```diff
@@ -867,6 +906,55 @@ class SlideshowComponent extends SliderComponent {
     });
   }
 
+  applyAnimation(button = 'next') {
+    requestAnimationFrame(() => {
+      this.querySelectorAll('.announcement-bar__message').forEach((slide) => {
+        if (this.currentSlide === this.allAnnouncements.length && button === 'previous') {
+          slide.classList.add(`announcement-bar-slider--fade-out-next`);
+          setTimeout(() => {
+            slide.classList.remove(`announcement-bar-slider--fade-out-next`);
+          }, 200);
+          setTimeout(() => {
+            slide.classList.add(`announcement-bar-slider--fade-in-next`);
+          }, 200);
+          setTimeout(() => {
```

Comment:
Can some of these `setTimout` be avoided if we leverage [animation delay](https://developer.mozilla.org/en-US/docs/Web/CSS/animation-delay)?

Reply:
I tried but it didn't work for me. Maybe I missed something but that what I tried:

changed this
```
slide.classList.add(`announcement-bar-slider--fade-out-previous`);
setTimeout(() => {
  slide.classList.remove(`announcement-bar-slider--fade-out-previous`);
  slide.classList.add(`announcement-bar-slider--fade-in-previous`);
}, this.delay);
setTimeout(() => {
  slide.classList.remove(`announcement-bar-slider--fade-in-previous`);
}, this.delay * 2);
```
to this
```
slide.classList.add(`announcement-bar-slider--fade-out-previous`);
slide.classList.add(`announcement-bar-slider--fade-in-previous`);
setTimeout(() => {
  slide.classList.remove(`announcement-bar-slider--fade-out-previous`);
  slide.classList.remove(`announcement-bar-slider--fade-in-previous`);
}, this.delay * 2);
```
and added `animation-delay: 200ms` to `announcement-bar-slider--fade-in-previous`

Reply:
Ok I think we can keep the `setTimeout`. One other question is why we need to apply classes on every slide on every button click? Could we only need to apply animation on the outgoing and incoming slide?